qiskit/visualization/matplotlib.py

```diff
@@ -276,16 +226,72 @@ def _registers(self, creg, qreg):
 
     @property
     def ast(self):
+        """AST getter"""
         return self._ast
 
+    def _load_style(self, style):
+        current_style = DefaultStyle().style
+        style_name = 'default'
+        config = user_config.get_config()
+        if style is False:
+            style_name = 'bw'
+        elif isinstance(style, dict) and 'name' in style.keys():
+            style_name = style['name']
+        elif isinstance(style, str):
+            style_name = style
+        elif config:
+            style_name = config.get('circuit_mpl_style', 'default')
```

Comment:
Maybe raising in case of `else`?
```suggestion
            style_name = config.get('circuit_mpl_style', 'default')
     else:
           raise something
```

Reply:
sounds good to warn and continue with default if `style` is not a `str` and there is no `config`.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5223/commits/db411c8f887c9f71a5bcd2a9455f94ab46115917